user-image/d8extension.bash

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cd tmp
```

Comment:
This should be /tmp.

I'd also add `set -e` as the second line in the script so that the script fails if any line in the script errors.